python/GafferUI/SpreadsheetUI/_Metadata.py

```diff
@@ -208,7 +208,10 @@ def __defaultCellMetadata( plug, key ) :
 for key in [
 	"spreadsheet:columnLabel",
 	"plugValueWidget:type",
-	"presetsPlugValueWidget:allowCustom"
+	"presetsPlugValueWidget:allowCustom",
+	"ui:scene:acceptsSetName",
+	"ui:scene:acceptsSetNames",
+	"ui:scene:acceptsSetExpression"
```

Comment:
It's a pity to bake knowledge of GafferSceneUI into the spreadsheet code like this, but I think it's the right option for now. Longer term I think it may make sense to have a mechanism to allow external components to opt-in as necessary, but we'll have a better idea of what that should look like once we have a few more examples.